validator/testcases/javascript/predefinedentities.py

```diff
@@ -64,6 +101,12 @@
 ]
 
 
+def is_shared_scope(t, r=None, rn=None):
```

Comment:
I don't know what a "shared scope" is. Can you please explain that in the docstring?